docs/docusaurus/docs/bpmn/ch07b-BPMN-Constructs.md

```diff
@@ -3726,7 +3726,7 @@ A while back, we also introduced a new type of execution listener, the org.flowa
 
     </flowable:executionListener>
 
-Since Flowable 6.8.0 scriptable execution listeners became a first-class citizen using a more compact syntax, avoiding
+Scriptable execution listeners became a first-class citizen using a more compact syntax, avoiding
```

Comment:
Wouldn't this slightly changed wording read better:
```suggestion
Scriptable execution listeners are a first-class citizen using a more compact syntax, avoiding
```

Reply:
This paragraph is an extension of the paragraph, starting with 
"A while back, we also introduced a new type of execution listener, the org.flowable.engine.impl.bpmn.listener.ScriptExecutionListener. ". It would read a bit awkward in this context.

Reply:
I see; thanks for the clarification.   It *was* just a suggestion 😏